### ISSUE TITLE
Are you still here? =)

### ISSUE THREAD

**Issue by LukeSkywalker92** [chanjr/rheedsim#1]
Hey,

are you still here? =)
If yes i would like to ask you some questions about rheedsim.

Best
Luke

**Comment by chanjr** [comment]
Yes, although I haven't worked directly on this code for quite a while now. I can try to answer any questions regardless.

**Comment by LukeSkywalker92** [comment]
Thank you for answering.
I have some questions to the json parameter files.

First, what exactly is the orientation matrix? Can it be used to simulate the rheed pattern along a specific crystal direction (e.g. [110] or [1-10]). I tried to figure it out myself, but i was not successful =)

Second, to simulate the rheed along a specific crystal direction I need to use an incident angle different to 0. Am I right, that the unit you use for incident angles is radians?

Thank you very much.

**Comment by chanjr** [comment]
The orientation matrix is how the crystal lattice orientation is stored with respect to the experiment geometry by applying the said transform to the lattice vectors defining the crystal. Specific crystal  directions can be more easily created by using the `Lattice.align_lattice` function in crystal.py, e.g. to align along the (111)[1-10] direction of some crystal

```
lattice = Lattice(...)
lattice.align_lattice([0, 0, 1], 1, 1, 1, direction = False) # aligns (1, 1, 1) plane with z axis of system geometry
lattice.align_lattice([1, 0, 0], 1, -1, 0, direction = True) # aligns [1, -1, 0] direction with x axis (beam direction)
```

The resulting orientation matrix would depend on the specific lattice system (cubic, hexagonal etc.)

Radians are the default for all the angle definitions, yes.
